media/js/firefox/whatsnew/whatsnew-id.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ (function() {
+     'use strict';
+
+     var client = Mozilla.Client;
+     var form = new Mozilla.SendYourself('download-firefox-rocket');
+     form.init();
+
+     function checkUpToDate() {
+         // bug 1419573 - only show "Your Firefox is up to date" if it's the latest version.
+         if (client.isFirefoxDesktop) {
+             client.getFirefoxDetails(function(data) {
+                 if (data.isUpToDate) {
+                     document.querySelector('.c-page-header').classList.add('is-up-to-date');
```

Comment:
When testing locally, the page header still looks a bit wonky:

![screenshot_2019-01-18 firefox lite](https://user-images.githubusercontent.com/400117/51380361-7a2dfa80-1b09-11e9-86ad-3203d4bdaf28.png)

Looking in your CSS, it looks like the JS class to apply should be:

```javascript
classList.add('show-up-to-date-message');
```

Even if someone views this page when their browser is not up to date, it would be nice if the Mozilla logo was still centered by default (but not a blocker).